packages/cli/test/fixtures/unit/commands/list/with-team/.vercel/project.json

```diff
@@ -0,0 +1,4 @@
+{
+  "projectId": "prj_Am19DF8JBL9g89tn4RdDVD59axFi",
+  "orgId": "team_MtLD9hKuWAvoDd3KmiHs9zUg"
+}
```

Comment:
These look like real project/org ids 🤔 

I would prefer the test is mocked using fake IDs (or use an integration test to run vc list after creating deployments)